bindings/python/test/test_bson.py

```diff
@@ -103,8 +106,8 @@ def test_simple(self):
 
         schema = Schema({'_id': ObjectId,
                          'data': int64(),
-                         'title': pyarrow.string() })
+                         'valid': pyarrow.bool_() })
```

Comment:
Can we change this to a type that we never plan to support?

Reply:
Done